SolrNet/Schema/SolrSchemaParser.cs

```diff
@@ -37,12 +37,16 @@ public class SolrSchemaParser : ISolrSchemaParser {
 
             var schemaElem = solrSchemaXml.Element("schema");
 
-            foreach (var fieldNode in schemaElem.XPathSelectElements("types/fieldType|types/fieldtype")) {
+            foreach (var fieldNode in schemaElem.XPathSelectElements("types/fieldType|types/fieldtype|fieldtype|fieldType")) {
```

Comment:
Could you back up this code with one or two unit tests?
See `SolrSchemaParserTests.cs`

Reply:
I'll do, but give me time until next week

Reply:
I did the requested changes. I hope I did the merge well. Was my first time. Do I have to mark your requests as solved somewhere?